geomesa-accumulo-bundle/geomesa-accumulo-processors/src/test/scala/org/geomesa/nifi/processors/accumulo/PutGeoMesaAccumuloTest.scala

```diff
@@ -323,4 +323,72 @@ class PutGeoMesaAccumuloTest extends LazyLogging {
       ds.dispose()
     }
   }
+
+  @Test
+  def testRecordIngestFlowFileAttributes(): Unit = {
+    val catalog = s"${root}RecordIngestAttributes"
+    val runner = TestRunners.newTestRunner(new PutGeoMesaAccumuloRecord())
+    try {
+      dsParams.foreach { case (k, v) => runner.setProperty(k, v) }
+      runner.setProperty(AccumuloDataStoreParams.CatalogParam.key, catalog)
+      val service = new AvroReader()
+      runner.addControllerService("avro-record-reader", service)
+      runner.setProperty(service, SchemaAccessUtils.SCHEMA_ACCESS_STRATEGY, "embedded-avro-schema")
+      runner.enableControllerService(service)
+      runner.setProperty(Properties.RecordReader, "avro-record-reader")
+      runner.setProperty(Properties.TypeName, "${type-name}")
+      runner.setProperty(Properties.FeatureIdCol, "${id-col}")
+      runner.setProperty(Properties.GeometryCols, "${geom-cols}")
+      runner.setProperty(Properties.GeometrySerializationDefaultWkt, "WKB")
```

Comment:
I changed the labels to just be the simple text 'WKB'/'WKT', since it supports nifi expressions now